kubecost/values.yaml

```diff
@@ -54,58 +54,24 @@ global:
 
   ## Kubecost Alerting
   ## Ref: https://www.ibm.com/docs/en/kubecost/self-hosted/2.x?topic=ui-alerts
-  ##
   notifications:
     # alertConfigs:
-    #   frontendUrl: http://localhost:9090  # Optional
+    #   frontendUrl: http://localhost:9090/  # Optional
```

Comment:
Please provide a short sentence of what this does and why/when a user would wish to configure it.